recipes/default.rb

```diff
@@ -24,23 +26,25 @@
 
 case node['platform']
 when 'ubuntu'
-  package 'cron' # /etc/cron.d is required.
-  if node['platform_version'].to_i == 12
-    node.default['cron']['emulate_cron.d'] = true
```

Comment:
This is not necessary for Ubuntu 12, and actually results in double execution of everything in `/etc/cron.d/`